framework/core/locale/core.yml

```
@@ -550,6 +550,7 @@ core:
       dependent_extensions_message: "Cannot disable {extension} until the following dependent extensions are disabled: {extensions}"
       extension_initialiation_failed_message: "{extension} failed to initialize, check the browser console for further information."
       generic_message: "Oops! Something went wrong. Please reload the page and try again."
+      generic_cross_origin_message: "Oops! Something went wrong (cross origin request). Please reload the page and try again."
```

If we choose to show this to all visitors, I think the CORS message should either be at the end of the text, or flow with the other text.

```suggestion
      generic_cross_origin_message: "Oops! Something went wrong during a cross-origin request. Please reload the page and try again."
```

```suggestion
      generic_cross_origin_message: "Oops! Something went wrong. Please reload the page and try again. (cross-origin request)"
```

---

I like the first proposal ("during").

I fear that if it is at the end, there's more chance it will be omitted during a support request.

I'll wait a bit before making the change to hear if there's other feedback (or someone from the core team can do it).

---

I like the first proposal as well.